### PR TITLE
chore: In app ratings modal and profile picture picture sheet UI improvements

### PR DESCRIPTION
**Pull request by saeedbashir**
### Description

Jira Ticket: [LEARNER-10307](https://2u-internal.atlassian.net/browse/LEARNER-10307)

| Before  | After |
| ------------- | ------------- |
| profile  | profile  |
| review1  | review1  |
| review2  | review2  |

Note: This change will not affect openedX theme.

### CODE REVIEW

File: Core/Core/View/Base/AppReview/Elements/AppReviewButton.swift [hunk 1]
```diff
@@ -18,26 +18,42 @@ struct AppReviewButton: View {
...
                 .padding(.vertical, 9)
         }.fixedSize()
-            .background(isActive
-                        ? Theme.Colors.accentColor
-                        : Theme.Colors.cardViewStroke)
+                .background(
+                    Theme.Shapes.buttonShape
+                        .fill(
+                            isActive ? Theme.Colors.accentColor
+                            : Theme.Colors.cardViewStroke
```
Comment (rnr) [inline Core/Core/View/Base/AppReview/Elements/AppReviewButton.swift:46]: do we need next line here?

**Review by rnr — APPROVED** [review]

File: Core/Core/View/Base/AppReview/Elements/AppReviewButton.swift (hunk 1 above)
Comment (saeedbashir) [reply Core/Core/View/Base/AppReview/Elements/AppReviewButton.swift:46]: I guess yes, as the newline is added here as well `.fill(`

Comment (rnr) [reply Core/Core/View/Base/AppReview/Elements/AppReviewButton.swift:46]: My question is about `: Theme.Colors.cardViewStroke` - do we need to have this on new line?

Comment (rnr) [reply Core/Core/View/Base/AppReview/Elements/AppReviewButton.swift:46]: I think splitting ternary operators makes sense if it exceeds the string length limit

**Review by shafqat-muneer — APPROVED** [review]

**Pull request merged**